ext/standard/tests/general_functions/bug44667.phpt

```diff
@@ -1,7 +1,10 @@
 --TEST--
 Bug #44667 (proc_open() does not handle pipes with the mode 'wb' correctly)
 --SKIPIF--
-<?php if (!is_executable('/bin/cat')) echo 'skip cat not found'; ?>
+<?php
+if (!is_executable('/bin/cat')) echo 'skip cat not found';
```

Comment:
any difference between `echo 'skip not found';` and `die 'skip proc_open..'` ?

Reply:
@staabm The only one that I know is, with we run the generated `test.sh`, it would break. `die` would return the message and kill the script.

Reply:
Should echo or die be unified ?

Reply:
@staabm Sure it does, `SKIPIF` should use `die`. [QA recommends this](http://qa.php.net/phpt_details.php#skipif_section) already.

I'm waiting for this one, related to unifying requirement for `skipif.inc`, then we can unify this as well :smile: